ui/src/progress.tsx

```diff
@@ -37,6 +38,15 @@ import React from 'react'
  * Also, combine steps of a complex operation into one total bar to avoid “rewinding” the bar.
  * Instead change the label to reflect the change if necessary. Bars moving backwards reduce confidence in the service.
 */
+
+const
+  css = stylesheet({
+    leftAlign: {
+      position:'absolute',
+      marginLeft:0
+    },
+  })
```

Comment:
Hm.. I think this would not work if wrapped in `ui.inline`.

```py
ui.inline(items=[ui.progress(label='Spinner Progress', type='spinner')], justify='center'),
```

^^ Should be perfectly centered.

Let's use `styles` prop on `Fluent.Spinner` instead. `Fluent.Spinner styles={{ root: { justifyContent: 'auto' } }}` should do the trick. Please test it out.

Reply:
okay ill start working on the styles and description @mturoci